backend/src/main/resources/run.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE artshare.art_item_info
+    ADD FULLTEXT(name, description, category, labels);
+
+ALTER TABLE artshare.event_info
+    ADD FULLTEXT(title, description, category, labels);
+
+ALTER TABLE artshare.discussion_ost
```

Comment:
this should be "artshare.discussion_post"

Reply:
Done